packages/ui/src/common/queries/blocks.graphql

```diff
@@ -4,4 +4,5 @@ fragment BlockFields on Block {
   number
   timestamp
   network
+  createdAt
```

Comment:
~I'm sorry, I forgot about that. For block, it looks like we have to use `timestamp`. That's because the block's timestamp is related to the actual time a block was mined, while `createdAt` is the timestamp of the DB record.~ Fortunately, I'm wrong here :sweat\_smile: 

It looks like all the [timestamps are related to the block time](buy_membership_without_referrer), which makes more sense.

This however, makes a little mess in handling :) I'd prefer to use a Block's timestamp here as we already are going to use `Block` instance to display that block-date component.